secure-ios-app/mobile-services.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "1.0",
+    "clusterName": "192.168.64.74:8443",
+    "namespace": "myproject",
+    "clientId": "example_client_id",
+    "services": [
+    {
+        "id": "keycloak",
+        "name": "keycloak",
+        "type": "keycloak",
+        "url": "https://www.mocky.io/v2/5a6b59fb31000088191b8ac6",
+         "config": {
+             "auth-server-url": "https://keycloak.security.feedhenry.org/auth",
+             "clientId": "client-app",
```

Comment:
@twnolan We can remove `clientId` from here as it's no longer included in the configuration since https://github.com/aerogearcatalog/keycloak-apb/pull/66.